src/dcos_e2e/cluster.py

```diff
@@ -117,13 +117,20 @@ def _wait_for_dcos_diagnostics(self) -> None:
         """
         for node in self.masters:
             node.run(
-                args=['/opt/mesosphere/bin/dcos-diagnostics', '--diag'],
+                args=[
```

Comment:
Why these changes? To support 1.9? If so - can this be separate and part of https://jira.mesosphere.com/browse/DCOS_OSS-2299?

Reply:
Required for resilience tests. Yes, this is the only change needed to support 1.9 with the AWS backend.